coverage-cop/action.yml

```diff
@@ -15,92 +15,93 @@ inputs:
 runs:
   using: "composite"
   steps:
-      - env:
-          stepName: Install Dependencies
-          bashPass: \033[32;1mPASSED -
-          bashInfo: \033[33;1mINFO -
-          bashFail: \033[31;1mFAILED -
-          bashEnd:  \033[0
-        shell: bash
-        name: ${{ env.stepName }}
-        run: |
-          # ${{ env.stepName }}
-          echo -e "::group::${{ env.bashInfo }} ${{ env.stepName }} ${{ env.bashEnd }}"
-          sudo apt-get install lcov fd-find
-          echo -e "::endgroup::"
-          echo -e "${{ env.bashPass }} ${{env.stepName }} ${{ env.bashEnd }}"
+    - env:
+        stepName: Install Dependencies
+        bashPass: \033[32;1mPASSED -
+        bashInfo: \033[33;1mINFO -
+        bashFail: \033[31;1mFAILED -
+        bashEnd:  \033[0
+      name: ${{ env.stepName }}
+      shell: bash
+      run: |
+        # ${{ env.stepName }}
+        echo -e "::group::${{ env.bashInfo }} ${{ env.stepName }} ${{ env.bashEnd }}"
+        sudo apt-get install lcov fd-find
+        echo -e "::endgroup::"
+        echo -e "${{ env.bashPass }} ${{env.stepName }} ${{ env.bashEnd }}"
 
-      - env:
-          stepName: Check Line and Branch Coverage
-          bashPass: \033[32;1mPASSED -
-          bashInfo: \033[33;1mINFO -
-          bashFail: \033[31;1mFAILED -
-          bashEnd:  \033[0
-        name: ${{ env.stepName }}
-        id: action-check-line-and-branch-coverage
-        shell: bash
-        run: |
-          # ${{ env.stepName }}
+    - env:
+        stepName: Check Line and Branch Coverage
+        bashPass: \033[32;1mPASSED -
+        bashInfo: \033[33;1mINFO -
+        bashFail: \033[31;1mFAILED -
+        bashEnd:  \033[0
+      name: ${{ env.stepName }}
+      id: action-check-line-and-branch-coverage
+      shell: bash
+      run: |
+        # ${{ env.stepName }}
 
-          # Print the received code cov report.
-          # TODO: The way it grabs the line/branch coverage is a little complicated
-          # I'd like to see if this can be done simpler.
-          echo -e " ${{ env.bashInfo }} Received LCov Report: ${{ inputs.line-coverage-min }} ${{ env.bashEnd}}"
-          lcov --list --rc lcov_branch_coverage=1 ${{ inputs.coverage-file }}
-          LINE_COVERAGE=$(lcov --list ${{ inputs.coverage-file }} | tail -n 1 | cut -d '|' -f 2 | sed -n "s/\([^%]*\)%.*/\1/p")
-          BRANCH_COVERAGE=$(lcov --rc lcov_branch_coverage=1 --list ${{ inputs.coverage-file }} | tail -n 1 | cut -d '|' -f 4 | sed -n "s/\([^%]*\)%.*/\1/p")
-          RESULT=0
+        # Print the received code cov report.
+        # TODO: The way it grabs the line/branch coverage is a little complicated
+        # I'd like to see if this can be done simpler.
+        echo -e " ${{ env.bashInfo }} Received LCov Report: ${{ inputs.coverage-file }} ${{ env.bashEnd}}"
```

Comment:
Fix bug where didn't print the actual coverage report file, just the coverage line
Example of this bug can be found [here](https://github.com/FreeRTOS/coreMQTT/actions/runs/6475640446/job/17583009990#step:6:117)